test/checks/lists/dlitem.js

```diff
@@ -21,6 +21,12 @@ describe('dlitem', function () {
 		assert.isFalse(checks.dlitem.evaluate.apply(null, checkArgs));
 	});
 
+	it('should fail if the dlitem has a parent <dl> with a changed role', function(){
```

Comment:
Should be "should fail if the **dt element** has a parent <dl> with a changed role"